src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { isMarried, partnerName, name, age, sex } = person;
+
+  const isPersonMarried = isMarried ? (
+    <p className="Person__partner">
+      {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
```

Comment:
Consider adding a check to ensure `partnerName` is defined before using it. This will prevent potential issues if `partnerName` is missing for a married person.